v2/pkg/controller/mpi_job_controller.go

```diff
@@ -115,6 +115,7 @@ const (
 
 	openMPISlotsEnv  = "OMPI_MCA_orte_set_default_slots"
 	intelMPISlotsEnv = "I_MPI_PERHOST"
+	mpichSlotsEnv    = "TODO"
```

Comment:
Does anyone know how to figure out what should be provided here?
Interestingly, the pi example works as is, i.e. with mpichSlotsEnv == TODO

Reply:
This should be the name of the environment variable for configuring the number of workers per host.

It works in the sense that it runs, because this would just set an environment variable called TODO. But it's not doing the intended outcome.

Reply:
In general, it's hard to overwrite the command line, because users might be using wrappers around `mpiexec`. That's why environment variables is better. Another option that openMPI and Intel MPI support is to set them in the hostfile.

Actually, it might be useful to use that in general because of this #445